phpBB/includes/functions_admin.php

```diff
@@ -2573,6 +2573,11 @@ function prune($forum_id, $prune_mode, $prune_date, $prune_flags = 0, $auto_sync
 		$sql_and .= ' AND topic_status = ' . ITEM_MOVED . " AND topic_last_post_time < $prune_date";
 	}
 
+	if ($prune_limit > 0)
+	{
+		$sql_and .= " LIMIT $prune_limit";
```

Comment:
Ok.

I'll try to get to it after I figure out another Cron bug.  This one seems unrelated to this patch at all as it appears to just crash before anything happens.  Its on queue_interval, but I disabled queue via blanking the functions in cron.  And since this is a fatal error its definitely a problem.

PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted (tried to allocate 72 bytes) in /*****/forums/phpbb/db/driver/driver.php on line 567